test/fixtures/approvals/v3_readme.approved.json

```diff
@@ -1,10 +1,10 @@
 {
   "exercise": {
     "track_id": "fake",
-    "language": "Fake",
     "slug": "three",
+    "language": "Fake",
     "name": "Three",
-    "readme": "# Three\n\nThis is three.\n\n* three\n* three again\n\nDo stuff.\n\n## Source\n\nThe internet. [view source](http://example.com/3)\n",
-    "blurb": "This is three."
+    "blurb": "This is three.",
+    "readme": "# Three\n\nThis is three.\n\n* three\n* three again\n\nDo stuff.\n\n## Source\n\nThe internet. [view source](http://example.com/3)\n"
   }
```

Comment:
On second thought--I see why the ordering changed. This should be excluded from the PR.